### ISSUE TITLE
Idea: Make configuration that does not require extra dependencies the default

### ISSUE THREAD

**Issue by jtremback** [Mistodon/config_struct#3]
I installed this library and immediately got some compile errors about the Serde and Toml crates. I fixed it by changing the config:

```rust
extern crate config_struct;

use config_struct::Options;

fn main() {
    config_struct::create_config("config.toml", "src/config.rs", &Options { derived_traits: vec!["Debug".to_string(), "Clone".to_string()] , generate_load_fns: false, ..Default::default() });
}
```

Can I suggest that these be made the default options to avoid this immediate compile error?

**Comment by mistodon** [comment]
Hey there - thanks for pointing out this issue. Your fix is probably the right way to go, but I'll think it over and do some testing.

**Comment by mistodon** [comment]
Took me long enough, but this should be fixed in the latest release (0.3.0).

There's additionally a `StructOptions::serde_default()` method to generate the old default (with serde support and loading functions).

Thanks again for reporting this! ^^
